cmd/thanos/query.go

```diff
@@ -217,6 +219,9 @@ func registerQuery(app *extkingpin.App) {
 			if feature == queryPushdown {
 				level.Warn(logger).Log("msg", "This option for --enable-feature is now permanently deprecated and therefore ignored.", "option", queryPushdown)
 			}
+			if feature == promqlExperimentalFunctions {
+				level.Warn(logger).Log("msg", "This option for --enable-feature is now permanently enabled and therefore a no-op.", "option", promqlExperimentalFunctions)
+			}
```

Comment:
We are adding a new experimental feature so it is not enabled by default. We should remove this.